server/api/sample_sizes.py

```diff
@@ -45,20 +42,12 @@ def sample_sizes_for_contest(contest: Contest):
             }
 
         elif election.audit_type == AuditType.BATCH_COMPARISON:
-            sample_results = rounds.cumulative_batch_results(election)
-            if round_one:
-                sample_results = {
-                    batch_key: {
-                        contest_id: {choice_id: 0 for choice_id in contest_results}
-                        for contest_id, contest_results in batch_results.items()
-                    }
-                    for batch_key, batch_results in sample_results.items()
-                }
+            cumulative_batch_results = rounds.cumulative_batch_results(election)
             sample_size = macro.get_sample_sizes(
                 election.risk_limit,
                 sampler_contest.from_db_contest(contest),
                 rounds.batch_tallies(election),
-                sample_results,
+                cumulative_batch_results,
```

Comment:
we need to do something similar to what you did for ballot polling here. and we need a test case to catch this. im happy to work on the test case - i have an idea of what to do

Reply:
I'll work on changing MACRO. Since this PR only pertains to ballot polling, do you mind if we make that a separate PR

Reply:
great, separate PR sounds good. but then we'll prob want to avoid introducing buggy behavior in this PR, so maybe just undo the changes to this bit?

Reply:
Done